drivers/sensor/CMakeLists.txt

```diff
@@ -86,6 +86,7 @@ add_subdirectory_ifdef(CONFIG_STTS751		stts751)
 add_subdirectory_ifdef(CONFIG_SX9500		sx9500)
 add_subdirectory_ifdef(CONFIG_TH02		    th02)
 add_subdirectory_ifdef(CONFIG_TMP007		tmp007)
+add_subdirectory_ifdef(CONFIG_TMP108        tmp108)
```

Comment:
Please replace spaces with tabs to be consistent with the rest of the file

Reply:
Ok, Should be done now